tests/shared/utils/test_validation.py

```diff
@@ -33,6 +33,17 @@ def test_validate_yaml_schema(file, schema):
     validation_utils.validate_yaml_schema(rasa.shared.utils.io.read_file(file), schema)
 
 
+@pytest.mark.parametrize(
+    "file, schema",
+    [
+        ("data/test_domains/actions.yml", DOMAIN_SCHEMA_FILE),
+    ],
+)
+def test_validate_yaml_schema_actions(file: Text, schema: Text):
```

Comment:
Looks good, although I remember we had a discussion about possibly adding some kind of integration test for this in the future. If you still think adding one is a good idea, can you please create a ticket in our backlog so we don't forget.

Reply:
Created: https://rasahq.atlassian.net/browse/ATO-750